orchestra/static/orchestra/common/components/checklist/partials/checklist.html

```diff
@@ -0,0 +1,21 @@
+<div class="checklist-wrapper">
+    <div ng-if="data.items.length" class="toggle-completed"
+         ng-click="data.hideCompleted = !data.hideCompleted">
```

Comment:
I can't understand from the code whether the default for `hideCompleted` is true or false.  i think it should be false.